README.md

```diff
@@ -38,7 +38,7 @@ The SDK has been tested on the following operating systems:
 * Windows
 * MacOS (Intel and Apple silicon)
 * Ubuntu Linux
-* WebAssembly (Wasm); NOTE: claim _generation_ is not available on Wasm.
+* WebAssembly (Wasm); NOTE: claim _generation_ is not available on WASM.
```

Comment:
According to https://webassembly.org/, the abbreviation is spelled "Wasm."  Let's use that spelling (capitalization).